### DISCUSSION TITLE
Is there any 'clear' answer from Infinity for the future of the project?

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1501] [Q&A] [answered]
Sorry if there's already an answer, but I looked around and I can't find any _clear_ answer from @Docile-Alligator regarding the future of Infinity like the Apollo project did.

Will Infinity become useless by next month and is there any potential modification that may be made to help with the future of this project?

I have some Reddit accounts and I'm not planning on using Reddit anymore if this app goes dark. I'd like to know what's the latest update and hope for a good outcome!

**Comment by ghost** [comment, marked as answer]
> Sorry if there's already an answer, but I looked around and I can't find any _clear_ answer from @Docile-Alligator regarding the future of Infinity like the Apollo project did.
> 
> Will Infinity become useless by next month and is there any potential modification that may be made to help with the future of this project?
> 
> I have some Reddit accounts and I'm not planning on using Reddit anymore if this app goes dark. I'd like to know what's the latest update and hope for a good outcome!

I'm stupid: https://www.reddit.com/r/Infinity_For_Reddit/comments/147bhsg/the_future_of_infinity/.

**Reply by adriandelgg** [reply]
Is there another link? That one points to 404
